src/Client.java

```diff
@@ -0,0 +1,122 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Client {
+
+    private String name;
+    private String surname;
+    private LocalDate dateBirth;
```

Comment:
birthday??

Reply:
Então, aqui eu pensei em data de nascimento, ai como no passaporte vem assim (date of birth), achei que era assim. Acha melhor trocar para birthday?

Reply:
ou birthday ou dateOfBirth igual vc comentou do passaporte. dateBirth é meio que uma mistura dos dois rs

Reply:
Entendido. Misturei tudo. rs. Vou melhorar isso.